app/lib/actions/scc_manager/subscribe_product.rb

```diff
@@ -43,6 +44,7 @@ class CreateProduct < Actions::Base
       def create_sub_plans
         product = ::Katello::Product.new
         product.name = input[:product_name]
+        product.gpg_key = ::Katello::GpgKey.find_by(id: input[:gpg_key], organization: input[:organization_id])
```

Comment:
What happens, if the gpg_key is not given / not set? will it still work?

Reply:
Yes, the gpg_key is optional and it chooses 'nil' by default.